README.md

```diff
@@ -22,7 +22,8 @@ jobs:
     secrets: inherit
 ```
 
-or with explicit secrets:
+or with secrets explicitly defined if the caller workflow is not from a Portworx repo 
```

Comment:
Does this really have anything to do with being a Portworx repo? Isn't it just about how the secret it named in that repo?

Reply:
The documentation says: `The inherit keyword can be used to pass secrets across repositories within the same organization, or across organizations within the same enterprise.`
I tried to call it from my forked private repo and it failed with the `inherit` keyword. I think it makes sense to do not pass all your secrets to some external workflow.

Reply:
Right. But I guess the secrets do need to have the same name for the workflow to be able to make any sense of them, right? We should probably list both of those conditions to prevent confusion.